### ISSUE TITLE
Logstash is not starting because it cannot find logstash-core/logstash-core

### ISSUE THREAD

**Issue by bhavyarm** [elastic/logstash#9900]
- Version: 5.6.11 BC1

Logstash fails to come up with this in the logs:

```
RuntimeError: Error loading logstash-core/logstash-core.jar file, cause: no such file to load -- logstash-core/logstash-core
   (root) at /usr/share/logstash/logstash-core/lib/logstash-core/logstash-core.rb:23
  require at org/jruby/RubyKernel.java:1040
   (root) at /usr/share/logstash/logstash-core/lib/logstash/runner.rb:1
  require at org/jruby/RubyKernel.java:1040
   (root) at /usr/share/logstash/lib/bootstrap/environment.rb:70
```

**Comment by bhavyarm** [comment]
cc @jsvd @LeeDr

**Comment by jsvd** [comment]
Looking at [the build log](https://internal-ci.elastic.co/view/Release/job/elastic+release-manager+release-staging/301/): 
```
 % grep :logstash-core:compileJava ~/Desktop/consoleText.txt -A 15
:logstash-core:compileJava#
# A fatal error has been detected by the Java Runtime Environment:
#
#  SIGSEGV (0xb) at pc=0x00007fcdd07c3968, pid=31646, tid=0x00007fcd6e309700
#
# JRE version: OpenJDK Runtime Environment (8.0_181-b13) (build 1.8.0_181-8u181-b13-0ubuntu0.16.04.1-b13)
# Java VM: OpenJDK 64-Bit Server VM (25.181-b13 mixed mode linux-amd64 compressed oops)
# Problematic frame:
# V  [libjvm.so+0xa56968]
```

**Comment by jasontedor** [comment]
Closed by #9902
